.babelrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
- [ ] [OPTIOAN] Good job 👏. Please don't forget to add screenshot of app to your readmde file 👍

Reply:
Thank you 😄